tardis/io/logger/panel_init.py

```diff
@@ -0,0 +1,104 @@
+import panel as pn
+import os
+import logging
+from IPython import get_ipython
+
+logger = logging.getLogger(__name__)
+
+VALID_MODES = ["moria", "notebook", "vscode", "vscode_noipy"]
```

Comment:
Are we keeping a "moria" mode?

Reply:
No this file was for testing- I will add this file in the logger widget PR.